docs/explanation/archive.rst

```diff
@@ -317,13 +317,13 @@ this is currently relevant for you.
 
 Because security updates contain fixes for :term:`Common Vulnerabilities and Exposures`
 (CVE), it is mission critical to distribute them as fast as possible to end users.
-Mirrors are a technical burdon in this case, because there is a delay between the
-synchronoization of a mirror and the primary Ubuntu package archive server.
+Mirrors are a technical burden in this case, because there is a delay between the
+synchronisation of a mirror and the primary Ubuntu package archive server.
 
-In the worst case a bad actor gets informed about a CVE and can utilize it, before
+In the worst case a bad actor gets informed about a CVE and can utilise it, before
```

Comment:
iirc, the style guide suggests avoiding "utilise" in favour of something simpler, like "use"